test/linearAlgebra/linearAlgebra.cpp

```diff
@@ -12,12 +12,14 @@
 #if NF_WITH_GINKGO
 
 template<typename ExecSpace>
-bool isNotKokkosThreads(ExecSpace ex)
+bool isNotKokkosThreads([[maybe_unused]] ExecSpace ex)
```

Comment:
```suggestion
bool isNotKokkosThreads(ExecSpace)
```
it should also eliminate the warning by deleting the variable name.
but I am okay with the keywords